setup_deb.sh

```diff
@@ -78,7 +78,8 @@ pip3 install --no-deps --upgrade \
    tqdm==4.64.0 \
    sacrebleu==2.3.1 \
    sentencepiece==0.1.97 \
-   tiktoken==0.3.3
+   tiktoken==0.3.3 \
+   ultralytics==8.0.75
```

Comment:
after running setup_deb ultralytics complains about lack of "psutil" :/ I suggest you add all "deps of deps" to the list, unless you want to see if AML on aarch64 is yet ready for standard requirements.txt file to handle python deps on the occasion of this PR